### DISCUSSION TITLE
Session Details overlap in Browserstack

### DISCUSSION THREAD

**Discussion by krististepanek** [webdriverio/webdriverio#13964] [Q&A]
Using the Browserstack service for WDIO, I see in the documentation that it defaults to automatically set the session name and updates the session status based on the assertion outcome: https://webdriver.io/docs/browserstack-service#sessionnameformat

However, if there is more than 1 test/`it` in a spec file, Browserstack uses the same session for all the tests. Therefore, Browserstack shows the session name and result of only the final test in the spec file because the name and status keeps getting overwritten.  How do I get separate sessions in Browserstack so that I can accurately see which tests ran and whether they passed?
![Screenshot 2024-12-10 at 3 14 54 PM](https://github.com/user-attachments/assets/173313fc-4e08-44a5-8381-fe10154ab960)

**Comment by christian-bromann** [comment]
> How do I get separate sessions in Browserstack so that I can accurately see which tests ran and whether they passed?

I recommend to reach out to BS support. The WebdriverIO core team is not maintaining the BS service and I am not sure how much the BS folks are observing the discussion thread.

**Comment by th0car** [comment]
@krististepanek - Have you tried using the skipSessionName option? It should affect the output. https://www.browserstack.com/docs/automate/selenium/organize-tests
Update:  Sorry - "sessionNameOmitTestTitle" was the option I was thinking of. See this: https://webdriver.io/docs/browserstack-service/#sessionnameomittesttitle
